std_msgs/Header header
uuv-teach-repeat/TrackPoint[] trackpoints